addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -1547,6 +1542,7 @@ var Class = require('web.Class');
 var core = require('web.core');
 var session = require('web.session');
 var time = require('web.time');
+const {Markup} = require('web.utils');
```

Comment:
Markup doesn't look used in this particular JS module.

Reply:
Good catch, looks like I forgot to remove the import when I lifted the marking out of this bit.